bindata/network/kuryr/004-daemon.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kuryr-cni
+  namespace: openshift-kuryr
+  annotations:
+    kubernetes.io/description: |
+      This DaemonSet launches the kuryr-daemon component.
+spec:
+  template:
+    metadata:
+      name: kuryr-cni
+      labels:
+        app: kuryr
+        component: network
+        type: infra
+        openshift.io/component: network
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      hostNetwork: true
+      serviceAccountName: kuryr
+      priorityClassName: system-node-critical
+      initContainers:
+      - name: install-cni-plugins
+        image: {{ .NodeImage }}
+        command:
+        - /bin/bash
+        - -c
+        - |
+          #!/bin/bash
+          set -ex
+          # Take over network functions on the node
+          rm -Rf /host-cni-bin/*
```

Comment:
This was a silly holdover from 3.11 - you shouldn't be removing other CNI binaries any more. Either remove only your plugin or just delete this line.